src/main/java/persistence/sql/mapping/ColumnData.java

```diff
@@ -7,22 +7,24 @@
 import java.lang.reflect.Field;
 
 public class ColumnData {
```

Comment:
```suggestion
public class Column {
```
`Column` 이라고 해도 충분하지 않을까요?

Reply:
jakarta.persistence.Column 예랑 겹쳐서 불편해서 해두었던건데 바꿔도 괜찮을것같아요!